bundles/org.openhab.ui/web/src/pages/settings/pages/sitemap/sitemap-edit.vue

```diff
@@ -176,15 +204,16 @@
 <script>
 import SitemapCode from '@/components/pagedesigner/sitemap/sitemap-code.vue'
 import WidgetDetails from '@/components/pagedesigner/sitemap/widget-details.vue'
-import MappingDetails from '@/components/pagedesigner/sitemap/mapping-details.vue'
+import AttributeDetails from '@/components/pagedesigner/sitemap/attribute-details.vue'
 import DirtyMixin from '../../dirty-mixin'
+import { isArray } from 'zrender/lib/core/util'
```

Comment:
This isn't right - zrender is a graphic library used in ECharts, importing it like that would screw up the webpack chunks and introduce an unwanted dependency.
You can just use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray.

Reply:
Oops, indeed, that is what I meant to use. Sorry about that.